### DISCUSSION TITLE
Futurenet Soroban single-transaction explorer

### DISCUSSION THREAD

**Discussion by heytdep** [stellar/sorobanathon#84] [General]
https://github.com/xycloo/soroban-explorer is a workspace with utils to explore futurenet and web apps that rely on those crates. Currently, I have built https://tx-explorer.xycloo.com/, which lets you explore a Soroban transaction (does all the matching and decoding stuff).

You can try it out by pasting a futurenet transaction hash into the input field.

**Comment by achallagundla** [comment]
Heya @heytdep ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 2500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
